powerauth-rest-client-spring/src/main/java/com/wultra/security/powerauth/rest/client/PowerAuthRestClient.java

```diff
@@ -232,22 +232,22 @@ public InitActivationResponse initActivation(InitActivationRequest request, Mult
     }
 
     @Override
-    public InitActivationResponse initActivation(String userId, Long applicationId) throws PowerAuthClientException {
+    public InitActivationResponse initActivation(String userId, String applicationId) throws PowerAuthClientException {
```

Comment:
This is not backward compatible.

Reply:
Yes, this is the trade off - we will document it and announce this on version release.